shared/signup/phone-number/verify.tsx

```diff
@@ -71,6 +71,18 @@ type BodyProps = {
   resendWaiting: boolean
 }
 export const VerifyBody = (props: BodyProps) => {
+  // build in a delay to resend
+  // disable first, then enable after 2s
+  const [resendDisabled, setResendDisabled] = React.useState(true)
+  const enableResend = Kb.useTimeout(() => setResendDisabled(false), 4000)
+  React.useEffect(() => enableResend(), [enableResend])
+  const {onResend: _onResend} = props
+  const onResend = React.useCallback(() => {
+    _onResend()
+    setResendDisabled(true)
+    enableResend()
+  }, [_onResend, enableResend])
```

Comment:
this should have `setResendDisabled` in it too, right?

Reply:
Looks like it's not needed. The linter is ambivalent if it's included but doesn't add it - I'll leave it out here.

> (The identity of the setCount function is guaranteed to be stable so it’s safe to omit.)

https://reactjs.org/docs/hooks-faq.html#what-can-i-do-if-my-effect-dependencies-change-too-often

Reply:
Looks like it's not needed. The linter is ambivalent if it's included but doesn't add it - I'll leave it out here.

> (The identity of the setCount function is guaranteed to be stable so it’s safe to omit.)

https://reactjs.org/docs/hooks-faq.html#what-can-i-do-if-my-effect-dependencies-change-too-often